images/linux/scripts/installers/android.sh

```diff
@@ -45,56 +45,54 @@ else
     exit 1
 fi
 
+# Add required permissions
```

Comment:
Looks like permissions are already set in lien 31

Reply:
fixed